### ISSUE TITLE
App crashing on getContactsAsync

### ISSUE THREAD

**Issue by anwersolangi** [mrousavy/react-native-jsi-contacts#1]
Hello, thanks for making this library and converting the library into JSI, I've installed the library and after running the example code my app crash immediately. Here is the code below I'm trying with:
```
  useEffect(() => {
    (async () => {
      const contacts = await getContactsAsync();
      console.log('COntacts => ', contacts);
    })();
  }, []);
```

**Comment by mrousavy** [comment]
Have you added the library to your MainActivity (step 2 at installation)

**Comment by anwersolangi** [comment]
> Have you added the library to your MainActivity (step 2 at installation)

Yes I already did, and also using your Vision Camera library and it is working fine, also getHashAsync() method is working fine

**Comment by mrousavy** [comment]
okay and what's the error message?
